### DISCUSSION TITLE
Check for Admin user in frontend template

### DISCUSSION THREAD

**Discussion by olivermerz** [pimcore/pimcore#17468] [Q&A / Support]
Is there a way to check for the logged-in Admin user in the frontend template? We would like to make certain template areas only visible to admin users in Editmode. The check via `\Pimcore\Tool\Admin::getCurrentUser()` in the controller and passing it to the template or `is_granted('ROLE_PIMCORE_ADMIN')` in the template does not seem to have any effect in the frontend context. Are there any other options here?

**Comment by markus-moser** [comment]
You need to include does areas into the admin firewall. The easiest way to do this is to prefix the URLs with /admin. If this is not possible you would need to change the pimcore_admin symfony security firewall settings...

**Reply by markus-moser** [reply]
Or do you just mean within the template of a document? In that case you have the twig variable `editmode` to check if you are in the frontend or backend. So this is really easy, e.g. like this:

https://github.com/pimcore/demo/blob/09388836c165f4d9d7a74a00a76debcc2cde37b2/templates/areas/text-accordion/view.html.twig#L24

**Reply by olivermerz** [reply]
Yes, I mean within the template of a document in Pimcore Admin/Backend. There we want to check in the template whether the logged in user is an admin user.

**Comment by designjpp** [comment]
Hello, 

this code works for me, I added it to the `defaultAction `in `ContentController ` and  `DefaultController`

```
 #[Template('content/default.html.twig')]
    public function defaultAction(RequestHelper $requestHelper): array
    {
        $param = $requestHelper->isFrontendRequestByAdmin();
        return ["isAdmin" => $param];
    }
```
